src/modules/universal-discovery/components/tab-content/create.panel.component.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import TabContentPanelComponent from './tab.content.panel.component';
+import FinderComponent from '../finder/finder.component';
+import CreateComponent from '../create/create.component';
+
+import './css/create.panel.component.css';
+
+const CreatePanelComponent = (props) => {
+    const wrapperAttrs = {className: 'c-create-panel'};
+    const maxHeight = props.maxHeight - 24;
+    const componentProps = Object.assign({}, props, { maxHeight, allowOnlyContainers: true });
+
+    if (!props.isVisible) {
+        wrapperAttrs.hidden = true;
+    }
+
+    return (
+        <div {...wrapperAttrs}>
+            <TabContentPanelComponent {...props}>
+                <div className="c-create-panel__first-step">
+                    <div className="c-create-panel__step-title">1) {props.labels.contentOnTheFly.chooseLangaugeAndContentType}</div>
```

Comment:
Using `ol > li` isn't the right way to go?

Reply:
So `li` would need to have inside a whole `create` or `finder` component? I think it's not the best structure.